.changeset/real-onions-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro-og-canvas': patch
```

Comment:
Let’s make this a minor version release to be safe — people relying on e.g. `^0.1.8` won’t automatically get updated to `^0.2.0`.

```suggestion
'astro-og-canvas': minor
```